meson.build

```diff
@@ -98,34 +99,44 @@ extended_warnings = [
 	'-Wdangling-else',
 	'-Wdouble-promotion',
 	'-Wduplicated-branches',
+	'-w24754', # Conversion rules for arithmetic operations in a comparison mean that one branch cannot be executed.
 	'-Wfloat-conversion',
 	# '-Wformat-overflow=2',
 	# '-Wformat-signedness',
 	'-Wformat-truncation',
 	'-Wformat=2',
+	'-w24774, # ‘<function>’ : format string expected in argument <position> is not a string literal '
```

Comment:
The `'` at the end here wants moving to just before the `,` - this should fix the Meson build failures.